clients/iOS/Grapevine/Grapevine/Event.swift

```diff
@@ -18,11 +18,9 @@ class Event: NSObject, Mappable {
     var eventId: Int!
     var feedId: Int!
     var timeProcessed: NSDate!
-    var startTimeNS: NSDate!
-    var startTime: CVDate!
-    var endTimeNS: NSDate!
-    var endTime: CVDate!
-    var repeatsWeekly: Int!
+    var startTime: EventTime!
```

Comment:
Is it typical to have so many implicitly unwrapped optionals? I understand when you import all that stupid Objective C stuff you can't avoid it, but is it necessary for ints and event times and other classes you make yourself? Just curious because I am not a Cocoa programmer.